sklearn/utils/testing.py

```diff
@@ -508,11 +537,10 @@ def uninstall_mldata_mock():
              '_SigmoidCalibration', 'VotingClassifier']
 
 
-
 def all_estimators(include_meta_estimators=False,
                    include_other=False, type_filter=None,
                    include_dont_test=False):
-                   
+
```

Comment:
cosmetics: no need for a blank line here. It can be removed entirely for consistency.